packages/kit/src/views/Staking/pages/ProtocolDetailsV2/index.tsx

```
@@ -163,17 +162,51 @@ function AlertSection({ alerts }: { alerts: IStakeEarnDetail['alerts'] }) {
+  protocolInfo?: IProtocolInfo;
 }) {
   const intl = useIntl();
   const handleClaim = useHandleClaim({
-    accountId: tokenInfo?.accountId || '',
+    accountId: protocolInfo?.earnAccount?.accountId || '',
     networkId: tokenInfo?.networkId || '',
   });
+  const tooltipElement = useMemo(() => {
+    if (rewards?.tooltip) {
+      switch (rewards.tooltip.type) {
+        case 'text':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)